marklogic-data-hub/src/main/resources/ml-modules/services/collector.xqy

```diff
@@ -46,12 +51,14 @@ declare function post(
   $input   as document-node()*
   ) as document-node()*
 {
-  debug:dump-env(),
+  perflog:logit('Collector.post',function() {
```

Comment:
I'd like to see a better name logged. at the minimum use the : vs the .

perflog:logit('collector:post')

but maybe

perflog:logit('GET => /v1/resources/collector')

I'm not sure what works best. I just want it to be pretty easy to track down the offending code if I find a performance issue.